src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
I have not idea why Matplotlib changes its mind - I think the data is still the same, but it adds padding at the top of the plot (effectively adding something to the upper axis limit, in this case).   It could just be a round off thing.    In any case, I don't think we can rely on 'auto' to do the right thing.

It is really different - at least 5 degrees?  

![frame0010fig1](https://cloud.githubusercontent.com/assets/1406529/7103251/765c9476-e05b-11e4-8192-e2a55a875d0b.png)

![frame0011fig1](https://cloud.githubusercontent.com/assets/1406529/7103254/7db3d590-e05b-11e4-9fd8-68efeccb7fc8.png)